packages/app-store/closecomothercalendar/test/lib/CalendarService.test.ts

```diff
@@ -1,4 +1,5 @@
 import CloseCom from "@calcom/lib/CloseCom";
+import type { CloseComAttendees, CloseComCalendarEvent } from "@calcom/lib/CloseCom";
```

Comment:
@leog heads up, not sure why these tests were failing but I had to create Close.com specific types